src/omemo/omemo.h

```diff
@@ -33,13 +33,18 @@
  *
  */
 #include <glib.h>
+#include <gcrypt.h>
 
 #include "ui/ui.h"
 #include "config/account.h"
 
 #define OMEMO_ERR_UNSUPPORTED_CRYPTO -10000
 #define OMEMO_ERR_GCRYPT             -20000
 
+#define OMEMO_AESGCM_NONCE_LENGTH 12
```

Comment:
should this be set to AES128_GCM_IV_LENGTH  ? In case they always should be the same?

Reply:
Yeah it's just that it is mentioned separatedly for the OMEMO media sharing document, so I thought it could be viewed as "separate" from the standard -- but I can set it to AES128_GCM_IV_LENGTH